lib/travis/build/script/cpp.rb

```diff
@@ -41,28 +41,30 @@ def use_directory_cache?
         private
 
           def compiler
-            config[:compiler].to_s
+            cxx
```

Comment:
That doesn't surprise me. The spec should be adapted to follow the changes to `cpp.rb`, but I don't know enough about Ruby to do so.